pkg/kubelet/kubelet.go

```diff
@@ -161,6 +162,9 @@ const (
 
 	// Minimum number of dead containers to keep in a pod
 	minDeadContainerInPod = 1
+
+	// The root huge page control directory in sysfs
+	hugepagesDirectory = "/sys/kernel/mm/hugepages/"
```

Comment:
The issue is that a cgroup subsystem of hugetlb is missing `/sys/fs/cgroup/hugetlb/` not a `/sys/kernel/mm/hugepages`.

Reply:
Fixed.

Reply:
This is correct for my system, /sys/fs/cgroup/hugetlb doesn't exist.

@PiotrProkop Do you know why the kubelet appears to work normally on the first start?

Reply:
Nope, @derekwaynecarr should know, because he wrote it 😄